sound/soc/intel/boards/Kconfig

```diff
@@ -599,6 +599,24 @@ config SND_SOC_INTEL_SOF_DA7219_MAX98373_MACH
 
 endif ## SND_SOC_SOF_JASPERLAKE
```

Comment:
Also fix all checkpatch issues, including the email issue

WARNING: From:/Signed-off-by: email address mismatch: 'From: "balamurugan.c" <bc@coresw01.gar.corp.intel.com>' != 'Signed-off-by: balamurugan.c <balamurugan.c@intel.com>'

Reply:
done